args.go

```diff
@@ -225,6 +225,10 @@ const (
 	ArgLoadBalancerAlgorithm = "algorithm"
 	// ArgRedirectHttpToHttps is a flag that indicates whether HTTP requests to the load balancer on port 80 should be redirected to HTTPS on port 443.
 	ArgRedirectHttpToHttps = "redirect-http-to-https"
+	// ArgEnableProxyProtocol is a flag that indicates whether PROXY protocol should be enabled on the load balancer
```

Comment:
Add a period at the end of the doc string so it's consistent with other entries.

```suggestion
	// ArgEnableProxyProtocol is a flag that indicates whether PROXY protocol should be enabled on the load balancer.
```